### PR TITLE
Added 3D-models for various tools and colliders for said models

### PR DESCRIPTION
**Pull request by Abb22TedStr**

### CODE REVIEW

**Review by tramvaj99 — CHANGES_REQUESTED** [review]
- [x] Could you add a collider to the floor of the room? Right now object fall through.
- [ ] The floor is too high, should always be below the player. 
- [ ] Is there a way to make the tools sit at the bottom of the basket instead of sitting on top of it?

**Comment by Abb22TedStr** [comment]
Ska merge:a Olles händer och se om Hampus gjorde ändringar till outlinen nästa vecka.

**Review by Abb-FilGal — CHANGES_REQUESTED** [review]
Change sax to scissor to comply with our choice to keep everything in english

**Review by Abb-FilGal — APPROVED** [review]

File: VR_RV/Assets/Scenes/Gameplay Scenes/Bare Bones/Scripts/OutlineCreator.cs
Comment (tramvaj99) [inline VR_RV/Assets/Scenes/Gameplay Scenes/Bare Bones/Scripts/OutlineCreator.cs:1]: This seems to be the old, unoptimized code for the outlinecreator. See now merged PR #41 for the optimized code.

**Review by tramvaj99 — COMMENTED** [review]
See comment on OutlineCreator.

File: VR_RV/Assets/Scenes/Gameplay Scenes/Bare Bones/Scripts/OutlineCreator.cs
Comment (Abb22TedStr) [reply VR_RV/Assets/Scenes/Gameplay Scenes/Bare Bones/Scripts/OutlineCreator.cs:1]: It seems my Visual Studio Code has trouble fetching the latest master branch from GitHub, so there might be more old code that I'm unaware of, sadly. I think this was the only file that wasn't up to date, however, and I've just copy-pasted the latest code from GitHub into my Visual Studio Code and commited it.

**Review by abbolisar — APPROVED** [review]
fint

**Pull request merged**